tracing/tests/support/field.rs

```diff
@@ -29,6 +30,8 @@ pub enum MockValue {
     Any,
 }
 
+impl Eq for MockValue {}
```

Comment:
This is only correct if we ensure no `NaN` values are present in the `F64` case.

Reply:
:+1: